src/main/java/kr/syeyoung/dungeonsguide/commands/CommandDgDebug.java

```diff
@@ -0,0 +1,525 @@
+                        if (partyMax < 2) {
+                            sender.addChatMessage(new ChatComponentText("§eDungeons Guide §7:: §cparty max can't be smaller than 2"));
+                            return;
+                        }
+
+                        PartyManager.INSTANCE.setMaxParty(partyMax);
+                        sender.addChatMessage(new ChatComponentText("§eDungeons Guide §7:: §fSuccessfully set partymax to §e" + PartyManager.INSTANCE.getMaxParty()));
+                    } catch (Exception e) {
+                        sender.addChatMessage(new ChatComponentText("§eDungeons Guide §7:: §c" + args[1] + " is not valid number."));
+                        return;
+                    }
+                }
+                break;
+            case "partyid":
+                sender.addChatMessage(new ChatComponentText("§eDungeons Guide §7:: §fInternal Party id: " + Optional.ofNullable(PartyManager.INSTANCE.getPartyContext()).map(PartyContext::getPartyID).orElse(null)));
+                break;
+            case "loc":
+                sender.addChatMessage(new ChatComponentText("§eDungeons Guide §7:: §fYou're in " + DungeonsGuide.getDungeonsGuide().getSkyblockStatus().getDungeonName()));
+                break;
+
+            case "saverun":
+                try {
+                    File f = Main.getConfigDir();
+                    File runDir = new File(f, "dungeonruns");
+                    runDir.mkdirs();
+
+                    File runFile = new File(runDir, UUID.randomUUID() + ".dgrun");
+
+                    DungeonContext dungeonContext = DungeonsGuide.getDungeonsGuide().getDungeonGodObject().getContext();
+                    if (dungeonContext == null) {
+                        sender.addChatMessage(new ChatComponentText("§eDungeons Guide §7:: §cCouldn't find dungeon to save!"));
+                        return;
+                    }
+                    DungeonEventHolder dungeonEventHolder = new DungeonEventHolder();
+                    dungeonEventHolder.setDate(dungeonContext.getInit());
+                    dungeonEventHolder.setPlayers(dungeonContext.getPlayers());
+                    dungeonEventHolder.setEventDataList(dungeonContext.getEvents());
+
+
+                    ObjectOutputStream oos = new ObjectOutputStream(Files.newOutputStream(runFile.toPath()));
+                    oos.writeObject(dungeonEventHolder);
+                    oos.flush();
+                    oos.close();
+                    sender.addChatMessage(new ChatComponentText("§eDungeons Guide §7:: §fSuccessfully saved dungeon run to " + runFile.getAbsolutePath()));
+                } catch (Exception e) {
+                    sender.addChatMessage(new ChatComponentText("§eDungeons Guide §7:: §cAn error occured while writing rundata " + e.getMessage()));
+                    e.printStackTrace();
+                }
+
+                break;
+            case "purge":
```

Comment:
DG purge is not really a debug command, it can still be used by users to refresh their cosmetic perms